soc/xtensa/esp32/linker.ld

```diff
@@ -248,17 +260,18 @@ SECTIONS
   #include "linker/common-rom/common-rom-logging.ld"
```

Comment:
Although not related to this PR, this is incorrect; it should be using `<>` and also `zephyr/`.

A clean-up of these should be done at some point.

Reply:
It works as is. Should it?